README.md

```diff
@@ -67,171 +57,154 @@ If you use the Habitat platform in your research, please cite the [Habitat](http
+    # Load embodied AI task (RearrangePick) and a pre-specified virtual robot
+    env = habitat.Env(
+        config=habitat.get_config("configs/tasks/rearrange/pick.yaml")
+      )
 
-1. Pull the habitat docker image: `docker pull fairembodied/habitat:latest`
+    observations = env.reset()
 
-1. Start an interactive bash session inside the habitat docker: `docker run --runtime=nvidia -it fairhabitat/habitat:v1`
+    # Step through environment with random actions
+    while not env.episode_over:
+        observations = env.step(env.action_space.sample())
+    ```
 
-1. Activate the habitat conda environment: `source activate habitat`
+    This script instantiates a Pick agent in ReplicaCAD scenes. The agent takes random actions and you should see something like:
+    ```bash
+      Agent acting inside environment.
+      Renderer: AMD Radeon Pro 5500M OpenGL Engine by ATI Technologies Inc.
+      Episode finished after 200 steps.
+    ```
 
-1. Benchmark a forward only agent on the test scenes data: `cd habitat-api; python examples/benchmark.py`. This should print out an output like:
-```bash
-2019-02-25 02:39:48,680 initializing sim Sim-v0
-2019-02-25 02:39:49,655 initializing task Nav-v0
-spl: 0.000
-```
+    See [`examples/register_new_sensors_and_measures.py`](examples/register_new_sensors_and_measures.py) for an example of how to extend habitat-lab from _outside_ the source code.
 
-## Details
-An important objective of Habitat Lab is to make it easy for users to set up a variety of embodied agent tasks in 3D environments. The process of setting up a task involves using environment information provided by the simulator, connecting the information with a dataset (e.g. PointGoal targets, or question and answer pairs for Embodied QA) and providing observations which can be used by the agents. Keeping this primary objective in mind the core API defines the following key concepts as abstractions that can be extended:
 
-* `Env`: the fundamental environment concept for Habitat. All the information needed for working on embodied tasks with a simulator is abstracted inside an Env. This class acts as a base for other derived environment classes. Env consists of three major components: a Simulator, a Dataset (containing Episodes), and a Task, and it serves to connects all these three components together.
+1. **Interactive testing**: Using you keyboard and mouse to control a Fetch robot in a ReplicaCAD environment:
+    ```bash
+    # Pygame for interactive visualization, pybullet for inverse kinematics
+    pip install pygame==2.0.1 pybullet==3.0.4
 
-* `Dataset`: contains a list of task-specific episodes from a particular data split and additional dataset-wide information. Handles loading and saving of a dataset to disk, getting a list of scenes, and getting a list of episodes for a particular scene.
+    # Interactive play script
+    python examples/interactive_play.py --never-end --add-ik
```

Comment:
```suggestion
    python examples/interactive_play.py --never-end
```
I think by default no ik so people don't have to install PyBullet.

Reply:
I tried playing around with both. It's very hard to control the arm without IK. Why the hesitation to pip install pybullet?

Reply:
Just an additional command they need to run to get things working, but I agree it's not a big deal to install pybullet.